packages/teams-js/src/internal/videoFrameTick.ts

```diff
@@ -0,0 +1,50 @@
+import { generateGUID } from './utils';
+
+export class VideoFrameTick {
+  private static readonly setTimeoutCallbacks: {
+    [key: string]: {
+      callback: () => void;
+      startedAtInMs: number;
+      timeoutInMs: number;
+    };
+  } = {};
+
+  public static setTimeout(callback: () => void, timeoutInMs: number): string {
+    const startedAtInMs = performance.now();
+    const id = generateGUID();
+    VideoFrameTick.setTimeoutCallbacks[id] = {
+      callback,
+      timeoutInMs,
+      startedAtInMs,
+    };
+    return id;
+  }
+
+  public static clearTimeout(id: string): void {
+    delete VideoFrameTick.setTimeoutCallbacks[id];
+  }
+
+  public static setInterval(callback: () => void, intervalInMs: number): void {
+    VideoFrameTick.setTimeout(function next() {
```

Comment:
Is there a reason why you're using `function next() {...` vs `next() => {...` ? The latter is used more throughout the codebase so I want to know if there's a reason for the inconsistency?

Reply:
There is no special reason, to use `function next() {...` so that I can pass it directly to the `setTimeout` function, using `next() => {...` means I need to create a `const next = () => {...` and pass it to `setTimeout(next,...)`